packages/solid/package.json

```diff
@@ -55,24 +55,25 @@
   "devDependencies": {
     "@lightningtv/l3-ui-theme-base": "workspace:*",
     "@lightningtv/vite-hex-transform": "^1.1.0",
-    "@storybook/addon-essentials": "^7.6.19",
-    "@storybook/addon-interactions": "^7.6.19",
-    "@storybook/addon-links": "^7.6.19",
-    "@storybook/blocks": "^7.6.19",
-    "@storybook/builder-vite": "^7.6.19",
-    "@storybook/html": "^7.6.19",
-    "@storybook/html-vite": "^7.6.19",
-    "@storybook/manager-api": "^7.6.19",
+    "@storybook/addon-docs": "^8.3.0",
```

Comment:
Probably not a huge deal, but do we want this one to be in sync with the rest?

Reply:
@latest woo, can fix